### PR TITLE
[UCSC-80]: QA fix for Interstitial block pattern

### PR DESCRIPTION
**Pull request by bjcooper**
Adds missing horizontal root padding.

Ticket: https://moderntribe.atlassian.net/browse/UCSC-83

### CODE REVIEW

**Review by MlKilderkin — APPROVED** [review]

**Comment by Herm71** [comment]
The Interstitial block pattern works perfectly on my Local Dev. Go ahead and merge!

File: src/scss/block-patterns/_interstitial.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* BLOCK PATTERN: Link List */
+
+.ucsc__interstitial {
```
[the remark this answers is not in the document]
Comment (Vinsanity) [reply src/scss/block-patterns/_interstitial.scss:3]: Let's move forward with it for now. Noting that we may need to review the global gutter size later when we start building pages with the new blocks.

**Review by Vinsanity — APPROVED** [review]
🟢

File: src/scss/block-patterns/_interstitial.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (bjcooper) [reply src/scss/block-patterns/_interstitial.scss:3]: For posterity (we talked about this in another thread), the spacing/padding properties in the block editor do not allow you to apply a _root_ spacing on an element. They use some spacing presets that are separate and different, so, to get the precise spacing this block calls for, we needed a bit of CSS.

**Pull request merged**